### PR TITLE
fixed binding issue #30

### PR DESCRIPTION
**Pull request by msaikaushik**
bind attribute works with property names.

### CODE REVIEW

**Comment by msaikaushik** [comment]
tried to fix it.. i guess now the bind property searches for attributes with any number of characters.

**Comment by eavichay** [comment]
I think you should avoid regexp as it is costly. Since this is a simple check, you want to check attribute name index-of 'bind:' equals 0

WDYT?

**Comment by msaikaushik** [comment]
@eavichay ,so you want it to work with attribute names with more than 0 characters right?

**Comment by eavichay** [comment]
Just check that the attribute *starts with* `bind:` instead of regular expression which will have better performance. For hundreds of nodes on the DOM, every one is looked up and simple string functions are faster than regexp, where it can be avoided.

**Comment by msaikaushik** [comment]
@eavichay , i've removed the regex for bind just as you suggested. Is it good or shall i improve it in any other way?

File: Slim.js [hunk 1]
```diff
@@ -956,8 +956,12 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
...
-    return (/^(bind):(\S*)/.exec(attr.nodeName)
-    );
+    if(attr.nodeName.indexOf("bind:") == 0) {
+      return attr.nodeName;
+    }    
+    else {
+      return null;
+    } 
   }, function (source, target, attribute, match) {
     var tAttr = match[2];
```
Comment (eavichay) [inline Slim.js:966]: `match` argument would be `bind:<something>` and therefore match[2] would be 'a'.

Did you run npm test before submitting?

**Review by eavichay — CHANGES_REQUESTED** [review]

**Pull request closed without merging**